discord-bot/commands/votefor.js

```diff
@@ -0,0 +1,162 @@
+const fetch = require("node-fetch");
+const {
+	API_BASE_URL,
+	VOTING_SERVER_API_BASE_URL,
+	MENTOR_GITHUB_IDS,
+	MLH_STAFF_GITHUB_IDS,
+	GITHUB_IDS_OF_FELLOWS_WITHOUT_DISCORD_IDS,
+} = require("../constants");
+const { githubToken } = require("../config");
+
+module.exports = {
+	name: "votefor",
+	description: "Receives, checks and casts votes for contest",
+	async execute(message, args) {
+		const arg = validateArgs(args, message);
+
+		if (!arg) return;
+
+		const voterDiscordId = message.author.id;
+		const recipientDiscordId = getDiscordIdFromMention(arg);
+
+		let voterGithubId = "";
+		let recipientGithubId = "";
+
+		// bot ID
+		if (recipientDiscordId === "717455864994463884") {
+			message.channel.send("Awww! But really, please vote for a human fellow.");
+			return;
+		}
+
+		try {
+			voterGithubId = await getGithhubIdFromDiscordId(voterDiscordId);
+			recipientGithubId = await getGithhubIdFromDiscordId(recipientDiscordId);
+
+			// TEMP
+			// voterGithubId = "ivov";
+			// recipientGithubId = "adfaris";
+		} catch (error) {
+			console.log(
+				`ERROR FOR ONE OF THESE DISCORD IDs:\n${voterDiscordId}\n${recipientGithubId}`
+			);
+			message.channel.send(
+				"Uh oh, one of those Discord IDs is missing! Paging @<716842220648661012>"
+			);
+			return;
+		}
+
+		// pre-checks
+
+		console.log(recipientGithubId);
+
+		// if (["ivov", "kendevops", "shu8"].includes(recipientGithubId)) {
+		// 	message.channel.send(
+		// 		"Oh that fellow is a voting event organizer! Please vote for another fellow!"
+		// 	);
+		// 	return;
+		// }
+
+		if ([MLH_STAFF_GITHUB_IDS].includes(recipientGithubId)) {

```

Comment:
Should this be inside `[` brackets if it's already an array? (and the other few)

Reply:
Nice catch! Yeah, these bracketed checks would have failed tomorrow during the test run. Will fix!